browser/brave_wallet/keyring_service_unittest.cc

```diff
@@ -2630,6 +2630,53 @@ TEST_F(KeyringServiceUnitTest, GetChecksumEthAddress) {
   EXPECT_EQ(GetChecksumEthAddress(&service, "hello"), "0x");
 }
 
+TEST_F(KeyringServiceUnitTest, SignTransactionByFilecoinKeyring) {
+  KeyringService service(json_rpc_service(), GetPrefs());
+  base::test::ScopedFeatureList feature_list;
+  feature_list.InitAndEnableFeature(
```

Comment:
`base::test::ScopedFeatureList feature_list{features::kBraveWalletFilecoinFeature}` is a bit simpler.

Reply:
done